commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
Thanks, I'll ask on StackOverflow.

Do you mind if I create a branch from your last commit and start working on the 'remove usage settings' bug in the meantime? I only have two days at home before I start traveling again on Tuesday so I was hoping to make the most of them. :)